common/src/main/scala/explore/common/TargetQueries.scala

```diff
@@ -121,7 +114,7 @@ object TargetQueries {
        t.magnitudes
       )
     )(s =>
-      TargetResult.name.set(s._1.value) >>>
+      TargetResult.name.set(s._1) >>>
```

Comment:
Does it work if we also change the getter too in line 111 to `t.name`? Maybe we can now use `explore.optics.disjointZip`.

Reply:
Good catch. I wasn't aware of `disjointZip` but it works. :)